lib/controller.rb

```diff
@@ -4,8 +4,16 @@
 require './lib/helpers/data_mapper'
 require 'pry'
 
-class SlowFood < Sinatra::Base
+class WeekThreeAssessment < Sinatra::Base
 
-  # Your code will go here.
+get '/' do
+@text = 'Kristoffer'
+erb :index
+  end
 
+  get '/name/:name' do
+  User.create(name:params[:name])
+
+
+  end
 end
```

Comment:
And that entire block ^^ needs to be indented one tab to fit "inside" the `class`.